Assets/MixedRealityToolkit.Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.CameraSystem
+{
+    /// <summary>
+    /// The Camera system controls the settings of the main camera.
+    /// </summary>
+    public class MixedRealityCameraSystem : BaseCoreSystem, IMixedRealityCameraSystem
+    {
+        private enum DisplayType
+        {
+            Opaque = 0,
+            Transparent
+        }
+
+        public MixedRealityCameraSystem(
+            IMixedRealityServiceRegistrar registrar,
+            BaseMixedRealityProfile profile = null) : base(registrar, profile)
+        {
+            cameraProfile = (MixedRealityCameraProfile)profile;
+        }
+
+        /// <summary>
+        /// Is the current camera displaying on an Opaque (AR) device or a VR / immersive device
+        /// </summary>
+        public bool IsOpaque
+        {
+            get
+            {
+                currentDisplayType = DisplayType.Opaque;
+#if UNITY_WSA
+                if (!UnityEngine.XR.WSA.HolographicSettings.IsDisplayOpaque)
```

Comment:
I wonder if you are building for ARCore / ARKit the display type should be transparent. Also...magic leap also should be transparent.

Reply:
I'm leaving that up to developers - they're now free to write a camera service that better accommodates AR settings.